pkg/controllers/workgenerator/controller.go

```diff
@@ -221,48 +221,47 @@ func (r *Reconciler) Reconcile(ctx context.Context, req controllerruntime.Reques
 	return controllerruntime.Result{}, syncErr
 }
 
-// updateBindingStatusWIthRetry sends the update request to API server with retry.
+// updateBindingStatusWithRetry sends the update request to API server with retry.
 func (r *Reconciler) updateBindingStatusWithRetry(ctx context.Context, resourceBinding *fleetv1beta1.ClusterResourceBinding) error {
 	// Retry only for specific errors or conditions
-	err := r.Client.Status().Update(ctx, resourceBinding)
-	if err != nil {
-		klog.ErrorS(err, "Failed to update the resourceBinding status, will retry", "resourceBinding", klog.KObj(resourceBinding), "resourceBindingStatus", resourceBinding.Status)
-		errAfterRetries := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
-			var latestBinding fleetv1beta1.ClusterResourceBinding
-			if err := r.Client.Get(ctx, client.ObjectKeyFromObject(resourceBinding), &latestBinding); err != nil {
-				return err
-			}
-			// Work generator is the only controller that updates conditions excluding rollout started which is updated by rollout controller.
-			if rolloutCond := latestBinding.GetCondition(string(fleetv1beta1.ResourceBindingRolloutStarted)); rolloutCond != nil {
-				found := false
-				for i := range resourceBinding.Status.Conditions {
-					if resourceBinding.Status.Conditions[i].Type == rolloutCond.Type {
-						// Replace the existing condition
-						resourceBinding.Status.Conditions[i] = *rolloutCond
-						found = true
-						break
-					}
-				}
-				if !found {
-					// Prepend the new condition if it wasn't found
-					resourceBinding.Status.Conditions = append([]metav1.Condition{*rolloutCond}, resourceBinding.Status.Conditions...)
+	//err := r.Client.Status().Update(ctx, resourceBinding)
+	//if err != nil {
+	//	klog.ErrorS(err, "Failed to update the resourceBinding status, will retry", "resourceBinding", klog.KObj(resourceBinding), "resourceBindingStatus", resourceBinding.Status)
```

Comment:
why comment those out?

Reply:
This was when I was debugging. Let me fix.